patches/arcade/0002-Remove-reference-to-Microsoft-CodeAnalysis-in-Packag.patch

```diff
@@ -1,4 +1,4 @@
-From da96eae5b54e03770ee4a915622d363b15c027b7 Mon Sep 17 00:00:00 2001
+From 934378165868bc05243911836150105cf733b30e Mon Sep 17 00:00:00 2001
```

Comment:
Can you avoid pushing changes to the patches that only change the "from {commit}" line to make diffs easier to read?

Reply:
Sure. No problem.